Specs/Scene/ModelSpec.js

```diff
@@ -1679,6 +1679,22 @@ defineSuite([
          });
     });
 
+    it('loads a gltf with uint32 indices', function() {
+        return loadModel(boxUint32Indices).then(function(m) {
+            verifyRender(m);
+            primitives.remove(m);
+        });
+    });
+
+    it('throws runtime error when loading a gltf with uint32 indices if OES_element_index_uint is disabled', function() {
+        var context = scene.context;
+        context._elementIndexUint = false;
+        return loadModel(boxUint32Indices).otherwise(function(e) {
+            expect(e).toBeDefined();
+            context._elementIndexUint = true;
```

Comment:
Save and restore the original value; it could be `false` on some systems in theory.